src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -76,7 +76,8 @@ Author: Daniel Kroening, kroening@kroening.com
   "(horn)(skip-loops):(apply-code-contracts)(model-argc-argv):" \
   "(show-threaded)(list-calls-args)(print-path-lengths)" \
   "(undefined-function-is-assume-false)" \
-  "(remove-function-body):"
+  "(remove-function-body):" \
+  "(remove-calls-nobody)"
```

Comment:
This flag is missing documentation, also if possible could you move the defines for it into `remove_calls_nobody.h` as described here: https://github.com/diffblue/cbmc/blob/master/CODING_STANDARD#L114